src/System.IO.Pipes/src/System/IO/Pipes/AnonymousPipeServerStream.Unix.cs

```diff
@@ -32,10 +32,10 @@ private void Create(PipeDirection direction, HandleInheritability inheritability
             // Create SafePipeHandles for each end of the pipe.  Which ends goes with server and which goes with
             // client depends on the direction of the pipe.
             SafePipeHandle serverHandle = new SafePipeHandle(
-                (IntPtr)fds[direction == PipeDirection.In ? Interop.libc.ReadEndOfPipe : Interop.libc.WriteEndOfPipe], 
+                (IntPtr)fds[direction == PipeDirection.In ? Interop.Sys.ReadEndOfPipe : Interop.Sys.WriteEndOfPipe], 
```

Comment:
Apologies, but my recent merge of #2952 is going to conflict with this (nothing significant, just moving a few things around).